scss/mixins/_buttons.scss

```
@@ -3,35 +3,35 @@
 // Easily pump out default styles, as well as :hover, :focus, :active,
 // and disabled options for all buttons
 
-@mixin button-variant($background, $border, $hover-background: darken($background, 7.5%), $hover-border: darken($border, 10%), $active-background: darken($background, 10%), $active-border: darken($border, 12.5%)) {
-  color: color-yiq($background);
+@mixin button-variant($background, $border, $color: color-yiq($background), $hover-background: darken($background, 7.5%), $hover-border: darken($border, 10%),  $hover-color: color-yiq($hover-background), $active-background: darken($background, 10%), $active-border: darken($border, 12.5%), $active-color: color-yiq($active-background)) {
```

Could we start to reformat our mixins with all these parameters? Starting to feel difficult to read :).

```suggestion
@mixin button-variant(
  $background,
  $border, $color: color-yiq($background),
  $hover-background: darken($background, 7.5%),
  $hover-border: darken($border, 10%), 
  $hover-color: color-yiq($hover-background),
  $active-background: darken($background, 10%),
  $active-border: darken($border, 12.5%),
  $active-color: color-yiq($active-background)
) {
```

---

I agree, that would be much easier to read. Was the `$color` argument, on line 8, intentional on the same line as `$border`? Or should it be moved to it's own newline?

---

> Or should it be moved to it's own newline?

Yeah, it should (I've updated @mdo 's comment). We should also do this with the outlined buttons.